lp-steam/templates/csgo/server.j2

```diff
@@ -42,4 +62,27 @@ sv_minupdaterate "{{ item.value.tickrate | default('128') }}"
 sv_minrate 80000
 sv_maxrate 0
 
-exec rules_{{ item.value.rules | default('esl5on5') }}.cfg
+{% if item.value.backup | default(false) %}
+mp_backup_round_auto 1 
+mp_backup_round_file backup 
+mp_backup_round_file_pattern {{ csgo_backup_round_file_pattern | default('%prefix%_%date%_%time%_%map%_round%round%.txt') }}
+{% endif %}
+
+{% if item.value.overtime | default(false) %}
+mp_overtime_enable 1
+mp_overtime_maxrounds 6
+mp_overtime_startmoney 10000
+{% endif %}
+
+sv_hibernate_when_empty 0
```

Comment:
We should never set this to 0, as it prevents servers from sleeping (they keep running at 128 tick). This is important for resource sharing on our machines and  makes multi-tenancy much more challenging. Disabling this.